Cargo.toml

```diff
@@ -16,7 +16,7 @@ lto = true
 opt-level = 3
 
 [[bin]]
-name = "cas"
+name = "nativelink"
```

Comment:
this change got me thinking, does [this comment](https://github.com/TraceMachina/nativelink/pull/573/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L1-L4) at the top of the file still hold? I feel these may have disappeared.

@allada do you know anything about that?